### DISCUSSION TITLE
unRAID docker: "unversioned API: requested URI not found"

### DISCUSSION THREAD

**Discussion by ZippoD** [qdm12/gluetun#854] [Q&A]
Anyone knows what this error is? This is what I am getting when I attempt to launch the webUI. Does gluetun even have WebUI? CLI commands seemed to be very limited:
/ # help
Built-in commands:
------------------
        . : [ [[ alias bg break cd chdir command continue echo eval exec
        exit export false fg getopts hash help history jobs kill let
        local printf pwd read readonly return set shift source test times
        trap true type ulimit umask unalias unset wait
/ #

**Comment by qdm12** [comment]
It's only a web API for now see https://github.com/qdm12/gluetun/wiki/HTTP-control-server

I'm working on a mobile UI but this is still a few months away.

**Reply by ZippoD** [reply]
I am not trying to access it on a mobile. I have the docker installed on unRAID and still fresh with it (and dockers). What do I need to modify to get the webUI to load?

**Reply by qdm12** [reply]
There is no web ui, only an API. You can use curl or Postman to send requests to it for now if you want.

**Reply by ZippoD** [reply]
Got it! Thank you. Should it be possible to route other Docker containers to it (e.g. thru the VPN)?

**Reply by qdm12** [reply]
Definitely, have a look at the Github wiki there is a page on how to connect containers to it.
